### ISSUE TITLE
Working with alignments crashes or hangs Firefox

### ISSUE THREAD

**Issue by digitalbio** [ncbi/icn3d#64]
I would like students to be able to compare a structure of the SARS-CoV-2 spike protein where the RBD is up and bound to ACE2 and a structure where the RBD is down.  Ideally, they could the A key to alternate between the structures and see how the RBD changes conformation.

1.  I found two structures that I could get alignment from in VAST+ - 7KQR and 7A94.  
2.  I tried to select chains and color them the same so we could see the movement.  This is shown in the image. 

But Firefox either crashes or hangs.  

<img width="1300" alt="Screen Shot 2021-11-21 at 4 10 43 PM" src="https://user-images.githubusercontent.com/6663433/142784792-8a2a8607-0282-45df-b3f1-da0059473813.png">

**Comment by jiywang3** [comment]
Could you try it without opening the "Seq. & Annotations" window? In this case, the sequence window used a lot of memory because of the number of residues.

**Comment by digitalbio** [comment]
We need to Sequences and Annotations window open in order to identify the protein chains.

**Comment by jiywang3** [comment]
You could use “Defined Sets” and “Aligned Sequences”, but avoid “Seq. & Annotations” when the sequences are long.

**Comment by digitaltodd** [comment]
Perhaps this is a larger project.  It seems to me that if we know there is a performance issue relate to sequence length, that iCn3D should be aware and not display the data. Users won't know when it is ok, or not ok, to view a sequence. Could long sequences be paged in some way to overcome the performance issue?

**Comment by jiywang3** [comment]
Yes. If only partial sequence is displayed, the number of html elements corresponding to the residues will decrease. This will use less memory.
 It also depends on the memory of the user's computer.
